tests/test_endl.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import print_function
+import os
+import io
+import warnings
+import sys
+from hashlib import md5
+
+import numpy as np
+from numpy.testing import assert_array_equal
+import nose
+from nose.tools import assert_equal
+
+from pyne.utils import QAWarning
+warnings.simplefilter("ignore", QAWarning)
+from pyne.endl import Library
+
+if sys.version_info[0] > 2:
+    from io import StringIO
+else:
+    from StringIO import StringIO
+
+if sys.version_info[0] > 2:
+    import urllib.request as urllib
+else:
+    import urllib
+
+
+def test_loadfile():
+    if not os.path.isfile('epdl97_eedl_Pb'):
+        urllib.urlretrieve(
+                "https://www-nds.iaea.org/epdl97/data/endl/eedl/za082000",
```

Comment:
This isn't a big deal but our style guide says to use single quotes for string literals (rather than double quotes) when it makes sense to do so. http://pyne.io/devsguide/style_guide.html#python-style-guide